src/frontend/src/pages/WorkPackageForm/CreateWorkPackageCRForm.tsx

```diff
@@ -0,0 +1,61 @@
+import { ChangeRequestReason, ChangeRequestType, validateWBS } from 'shared';
+import WorkPackageForm from './WorkPackageForm';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../utils/routes';
+import { useQuery } from '../../hooks/utils.hooks';
+import { CreateWorkPackageApiInputs } from '../../apis/work-packages.api';
+import { WPFormType, startDateTester } from '../../utils/form';
+import * as yup from 'yup';
+import { useCreateStandardChangeRequest } from '../../hooks/change-requests.hooks';
+import { useCurrentUser } from '../../hooks/users.hooks';
+
+const CreateWorkPackageCRForm: React.FC = () => {
+  const query = useQuery();
+  const wbsNum = query.get('wbs');
+  const history = useHistory();
+
+  if (!wbsNum) throw new Error('WBS number not included in request.');
+
+  const { mutateAsync } = useCreateStandardChangeRequest();
+
+  const currentUser = useCurrentUser();
+  const onSubmit = async (payload: CreateWorkPackageApiInputs) => {
+    const crPayload = {
+      submitter: currentUser,
+      wbsNum: validateWBS(wbsNum),
+      type: ChangeRequestType.Issue,
+      what: payload.name,
+      why: [{ explain: 'explination', type: ChangeRequestReason.Competition }],
+      proposedSolutions: [],
+      duration: payload.duration,
```

Comment:
these all need to go in a workPackageProposedChanges object that is of type WorkPackageProposedChangesCreateArgs